packages/devextreme/js/common/charts.d.ts

```diff
@@ -341,6 +338,43 @@ export type GradientColor = {
     color: string;
 };
 
+/**
+ * @docid
+ * @namespace DevExpress.common.charts
+ * @public
+ */
+export type Font = {
```

Comment:
"devextreme-internal-tools" fix is required

Reply:
fixed